lib/pf/SwitchFactory.pm

```diff
@@ -165,6 +179,36 @@ sub config {
     return \%temp;
 }
 
+sub getModule {
+    my ($type) = @_;
+    return (exists $TYPE_TO_MODULE{$type} ) ? $TYPE_TO_MODULE{$type} : undef;;
+}
+
+=item buildVendorsList
```

Comment:
I don't want to harass you with this but I really think you should stick to the old headers until we come up with the official header